cpp/src/Node.cpp

```diff
@@ -1836,6 +1847,22 @@ void Node::SetLocation
 }
 
 //-----------------------------------------------------------------------------
+// <Node::PacketDelay>
+// Set PacketDelay value for the node
+//-----------------------------------------------------------------------------
+void Node::SetPacketDelay
+(
+        uint32 const& _PacketDelay
+)
+{
+    	m_PacketDelay = _PacketDelay;
+    	// Notify the watchers of the value changes
+    	Notification* notification = new Notification( Notification::Type_NodeNaming );
```

Comment:
Hi, I made this change on Feb 17 (the last change requested), since then I'm waiting for an answer,
Thanks